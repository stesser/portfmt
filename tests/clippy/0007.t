OPTIONS_GROUP=		SOUND
OPTIONS_GROUP_SOUND=	SNDIO
OPTIONS_GROUP_VIDEO=	OPENGL

.include <bsd.port.mk>
<<<<<<<<<
# Options definitions
OPTIONS_GROUP
OPTIONS_GROUP_SOUND

# Unknown variables
# WARNING:
# The following variables were not recognized.
# They could just be typos or Portclippy needs to be made aware of them.
# Please double check them.
#
# Prefix them with an _ to tell Portclippy to ignore them.
#
# If in doubt please report this on portfmt's bug tracker:
# https://github.com/t6/portfmt/issues
OPTIONS_GROUP_VIDEO
