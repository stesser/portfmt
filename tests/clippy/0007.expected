# Options definitions
OPTIONS_GROUP
OPTIONS_GROUP_SOUND

# Unknown variables
# WARNING:
# The following variables were not recognized.
# They could just be typos or Portclippy needs to be made aware of them.
# Please double check them.
#
# Prefix them with an _ to tell Portclippy to ignore them.
#
# If in doubt please report this on portfmt's bug tracker:
# https://github.com/t6/portfmt/issues
OPTIONS_GROUP_VIDEO
